binance/client.py

```diff
@@ -3567,10 +3567,18 @@ def futures_leverage_bracket(self, **params):
         """
         return self._request_futures_api('get', 'ticker/leverageBracket', data=params)
 
+    def futures_account_transfer(self, **params):
+        """Execute transfer between spot account and futures account.
+
+        https://binance-docs.github.io/apidocs/futures/en/#new-future-account-transfer
+
+        """
+        return self._request_margin_api('post', 'futures/transfer', True, data=params)
+
     def transfer_history(self, **params):
         """Get future account transaction history list
 
-        https://binance-docs.github.io/apidocs/futures/en/#new-future-account-transfer
+        https://binance-docs.github.io/apidocs/futures/en/#get-future-account-transaction-history-list-user_data
```

Comment:
harmless but unrelated to PR

Reply:
Actually, I think it is related to the PR (if I remember correctly).  That comment links the POST endpoint documentation, but is surrounding the code for the GET endpoint (2 different actions).   I implemented the POST endpoint, and linked POST endpoint documentation and this specific line is me updating the GET function call to link the GET endpoint documentation.

Reply:
my bad, that makes sense

Reply:
No worries. Thanks for reviewing! :)

Reply:
@chrisghill How do you call the function? I have tried `client.futures_account_transfer(asset="USDT", amount="1")` but keep getting following error message: `Mandatory parameter 'type' was not sent, was empty/null, or malformed.`. What `type` argument should we provide?